src/sas/qtgui/Perspectives/Fitting/media/fitting_help.rst

```diff
@@ -932,32 +927,30 @@ Fitting with the following caveats:
    second and later *BatchPage* s that have the same name as a parameter in the
    first will show up allowing for plotting of that parameter across the
    models. The other parameters will not be available in the grid.
-  To choose the order of the fitpages in the fitting process, drag and drop rows in the
-  *Source choice for simultaneous fitting* table. 
+   To choose the order of the fitpages in the fitting process, drag and drop rows in the
+   *Source choice for simultaneous fitting* table.
 
 .. note::
-   a corralary of the above is that currently models created as a sum|multiply
+   a corollary of the above is that currently models created as a sum|multiply
    model will not work as desired because the generated model parameters have a
    p#_ appended to the beginning and thus radius and p1_radius will not be
    recognized as the same parameter.
 
+.. image:: combine_batch_grid.png
 
-..
-  .. image:: combine_batch_grid.png
+In the example shown above the data is a time series with a shifting peak.
+The first part of the series was fitted using the *broad_peak* model, while
+the rest of the data were fit using the *gaussian_peak* model. Unfortunately the
+time is not listed in the file but the file name contains the information. As
+described in :ref:`Grid_Window`, a column can be added manually, in this case
+called time, and the peak position plotted against time.
```

Comment:
Maybe this was intended to be a figure caption?  In which case changing `.. image::` above to `.. figure::` might be a better change?  Similarly for the next image.

Reply:
There are no other uses of 'figure' in that file, it being mostly screenshot walk-through. 

Looking at the current output and the git history more, there are several paragraphs and quite a few images that don't appear in the output at all, but I don't know if that is intentional https://github.com/SasView/sasview/commit/dbba13450dc275d14d77d7daaaaf60f96e3e5918.